docs/concepts/overview/working-with-objects/common-labels-annotations.md

```diff
@@ -0,0 +1,84 @@
+will allow objects to be described in a common manner that all tools can understand.
+
+In addition to supporting tooling, the common labels and annotations describe
+applications, in a generic sense, in a way that can queried or are otherwise useful
+when operating applications.
+{% endcapture %}
+
+{% capture body %}
+The recommended object metadata is broken apart so that some of it is labels while
+other parts are annotations. Labels are shorter but can be used when querying
+the Kubernetes API. Annotations allow for longer data that's not meant to be
+queried.
+
+The metadata is organized around the concept of an Application. Kubernetes is not
+a Platform as a Service and doesn't have or enforce a formal notion of an application.
+Instead, applications are informal, described with metadata, and the definition of what an
+application contains is loose.
+
+Note, these are recommended labels and annotations. They aid the use of managing
+applications but are not required for any core tooling to work.
+
+One common theme you'll notice about the labels and annotations is that they
+contain a prefix of `app.kubernetes.io`. Labels and annotations without a prefix
+are considered private to users. These are meant to be common labels that do not
+interfere with existing or custom user labels. To accomplish this a prefix is
+used.
+
+## Labels
+
+| Key                                 | Description           | Example  |
+| ----------------------------------- | --------------------- | -------- |
+| app.kubernetes.io/name              | The name of the application stack | wordpress |
+| app.kubernetes.io/version           | The current version of the running application (e.g., a semantic version, revision hash, etc.) | v1.2.3 |
+| app.kubernetes.io/component         | The component within the architecture | front-end |
+| app.kubernetes.io/component-name    | The name of a component within the application | mysql |
+| app.kubernetes.io/component-version | The current version of the component | 5.7.21 |
+| app.kubernetes.io/deploy-manager    | The tool being used to manage the operation of an application | helm |
+
+To illustrative these labels in action consider the following StatefulSet object
+containing them:
+
+```yaml
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  labels:
+    app.kubernetes.io/name: wordpress
+    app.kubernetes.io/version: "4.9.4"
+    app.kubernetes.io/component: database
+    app.kubernetes.io/component-name: mysql
+    app.kubernetes.io/component-version: "5.7.21"
+    app.kubernetes.io/deploy-manager: helm
+```
+
+## Annotations
+
+| Key                     | Description           | Example  |
+| ----------------------- | --------------------- | -------- |
+| app.kubernetes.io/usage | The location of an applications usage information (e.g., details for how it's being run by an organization). A preference is given to version specific information | https://example.com/foo/1.2/ |
```

Comment:
If you think it’s worth adding code style to the keys in the example above, repeat it here...